Ginger/Ginger/Actions/ApiActionsConversion/ApiActionsConversionWizard.cs

```diff
@@ -0,0 +1,248 @@
+
+Licensed under the Apache License, Version 2.0 (the "License")
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at 
+
+http://www.apache.org/licenses/LICENSE-2.0 
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS, 
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. 
+See the License for the specific language governing permissions and 
+limitations under the License. 
+*/
+#endregion
+
+using System;
+using System.Threading.Tasks;
+using amdocs.ginger.GingerCoreNET;
+using Amdocs.Ginger.Common;
+using Amdocs.Ginger.CoreNET;
+using Amdocs.Ginger.CoreNET.ActionsLib.ActionsConversion;
+using Ginger.Actions.ActionConversion;
+using Ginger.WizardLib;
+using GingerCore;
+using GingerCoreNET.SolutionRepositoryLib.RepositoryObjectsLib.PlatformsLib;
+using GingerWPF.WizardLib;
+
+namespace Ginger.Actions.ApiActionsConversion
+{
+    /// <summary>
+    /// This class is used to ApiActionsConversionWizard 
+    /// </summary>
+    public class ApiActionsConversionWizard : WizardBase, IActionsConversionProcess
+    {
+        public override string Title { get { return "Convert Webservices Actions"; } }
+
+        private ObservableList<BusinessFlowToConvert> mListOfBusinessFlow = null;
+        public ObservableList<BusinessFlowToConvert> ListOfBusinessFlow
+        {
+            get {
+                return mListOfBusinessFlow;
+            }
+            set
+            {
+                mListOfBusinessFlow = value;
+            }
+        }
+
+        public bool ParameterizeRequestBody { get; set; }
+
+        public bool PullValidations { get; set; }
+
+        public eModelConversionType ModelConversionType 
+        { 
+            get
+            {
+                return eModelConversionType.ApiActionConversion;
+            }
+        }
+
+        public Context Context;
+        public ObservableList<ConvertableActionDetails> ActionToBeConverted = new ObservableList<ConvertableActionDetails>();
+        ConversionStatusReportPage mReportPage = null;
+        ApiActionConversionUtils mConversionUtils = new ApiActionConversionUtils();
+
+        /// <summary>
+        /// Constructor
+        /// </summary>
+        /// <param name="context"></param>
+        public ApiActionsConversionWizard(Context context)
+        {
+            Context = context;
+            ListOfBusinessFlow = GetBusinessFlowsToConvert(); 
+
+            AddPage(Name: "Introduction", Title: "Introduction", SubTitle: "Webservices Actions Conversion Introduction", Page: new WizardIntroPage("/Actions/ApiActionsConversion/ApiActionsConversionIntro.md"));
+            AddPage(Name: "Select Business Flow's for Conversion", Title: "Select Business Flow's for Conversion", SubTitle: "Select Business Flow's for Conversion", Page: new SelectBusinessFlowWzardPage(ListOfBusinessFlow, context));
```

Comment:
we can't mention "Business Flow/s" as static string please use GingerDicser to support terminoglogy